src/main/java/org/reactome/orthoinference/EWASInferrer.java

```diff
@@ -207,6 +231,21 @@
 		return infEWASInstances;
 	}
 
+	/**
+	 * Retrieve all Wormbase gene names that match the homologue Id.
+	 * @param homologueId -- String homologue ID value from Orthopair file.
+	 * @return -- List<String> of Wormbase gene names that correspond to the homologue ID.
+	 */
+	private static List<String> getWormbaseGeneNames(String homologueId) {
+		List<String> geneNames = new ArrayList<>();
+		for (String geneId : ensgMappings.get(homologueId)) {
+			if (wormbaseMappings.get(geneId) != null) {
```

Comment:
This might be better as `if (wormbaseMappings.containsKey(geneId))`.  If it's true, then given the way that `wormbaseMappings` is populated, you're guaranteed to get a non-null list of gene names.

Reply:
I have quite a bit of code in other identifiers that needs to be switched to this too haha

Reply:
:) - it's not a vital change, but I like the intent expressed with `containsKey` method.